packages/server/test/support/fixtures/projects/e2e/cypress/integration/issue_4062_spec.coffee

```diff
@@ -0,0 +1,7 @@
+describe "issue 4062",  ->
+  afterEach ->
+    cy.get("element_does_not_exist", {timeout: 200}).first()
+
+  it "fails with correct chaingin afterEach err when both test and afterEach fail",  ->
```

Comment:
I know it is not that important, but you have a typo: "chaingin".

Reply:
😝 thanks